qiskit_experiments/curve_analysis/standard_analysis/oscillation.py

```diff
@@ -123,3 +123,151 @@ def _evaluate_quality(self, fit_data: curve.FitData) -> Union[str, None]:
+                guess.constant_sinusoidal_offset`
+            bounds: [0, 1.5]
+
+        defpar \tau:
+            desc: Represents the rate of decay.
+            init_guess: Determined by :py:func:`~qiskit_experiments.curve_analysis.\
+                guess.oscillation_exp_decay`
+            bounds: [0, None]
+
+        defpar \rm freq:
+            desc: Oscillation frequency.
+            init_guess: Determined by :py:func:`~qiskit_experiments.curve_analysis.guess.frequency`
+            bounds: [0, 10 freq]
+
+        defpar \phi:
+            desc: Phase. Relative shift of the sinusoidal function from the origin.
+            init_guess: Set multiple guesses within [-pi, pi]
+            bounds: [-pi, pi]
+    """
+
+    __series__ = [
+        curve.SeriesDef(
+            fit_func=lambda x, amp, base, tau, freq, phi: curve.fit_function.cos_decay(
+                x,
+                amp=amp,
+                tau=tau,
+                freq=freq,
+                phase=phi,
+                baseline=base,
+            ),
+            plot_color="blue",
+            model_description=r"amp \exp(-x/tau) \cos(2pi freq x + phi) + base",
+        )
+    ]
+
+    def _generate_fit_guesses(
+        self, user_opt: curve.FitOptions
+    ) -> Union[curve.FitOptions, List[curve.FitOptions]]:
+        """Compute the initial guesses.
+
+        Args:
+            user_opt: Fit options filled with user provided guess and bounds.
+
+        Returns:
+            List of fit options that are passed to the fitter function.
+        """
+        curve_data = self._data()
+
+        user_opt.p0.set_if_empty(
+            amp=0.5,
+            base=curve.guess.constant_sinusoidal_offset(curve_data.y),
+        )
+
+        # frequency resolution of this scan
+        df = 1 / ((curve_data.x[1] - curve_data.x[0]) * len(curve_data.x))
+
+        if user_opt.p0["freq"] is not None:
+            # If freq guess is provided
+            freq_guess = user_opt.p0["freq"]
+
+            freqs = [freq_guess]
+        else:
+            freq_guess = curve.guess.frequency(curve_data.x, curve_data.y - user_opt.p0["base"])
+
+            # The FFT might be up to 1/2 bin off
+            if freq_guess > df:
+                freqs = [freq_guess - df, freq_guess, freq_guess + df]
+            else:
+                freqs = [0.0, freq_guess]
+
+        # Set guess for decay parameter based on estimated frequency
+        if freq_guess > df:
+            user_opt.p0.set_if_empty(
+                tau=-1
+                / curve.guess.oscillation_exp_decay(
+                    curve_data.x, curve_data.y - user_opt.p0["base"], freq_guess=freq_guess
+                )
+            )
+        else:
+            # Very low frequency. Assume standard exponential decay
+            user_opt.p0.set_if_empty(tau=-1 / curve.guess.exp_decay(curve_data.x, curve_data.y))
+
+        user_opt.bounds.set_if_empty(
+            amp=[0, 1.5],
+            base=[0, 1.5],
```

Comment:
These guesses don't work if input is level1  data.

Reply:
There was a general request of Oliver not to set bounds in the experiments (#161, #190). Oliver demonstrated how this biases T1 results, even with level 2 data. Following Oliver's issue, I removed bounds from T1 (#168). I also opened a PR that removes bounds from all the experiments (#320). I wanted to see if this was indeed the request before I proceed, but the discussion there never evolved.